unity-renderer/Assets/Scripts/MainScripts/DCL/DCLResources/ResourceManagerService.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections;
+using System.Collections.Generic;
+using Cysharp.Threading.Tasks;
+using DCL;
+using UnityEngine;
+
+public interface IResourceManagerService : IService
```

Comment:
Why do we need this? cannot we just use the keeper?

Reply:
Removed